src/fides/api/alembic/migrations/versions/c961528edfc6_add_website_connection_type.py

```diff
@@ -0,0 +1,108 @@
+"""add website connection type
+
+Revision ID: c961528edfc6
+Revises: 021166731846
+Create Date: 2025-01-27 19:20:20.236548
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "c961528edfc6"
+down_revision = "021166731846"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add 'datahub' to ConnectionType enum
```

Comment:
copypasta comment from the datahub migration

Reply:
you caught me! 😄